pyannote/audio/pipeline/speaker_diarization.py

```diff
@@ -266,6 +270,8 @@ def apply(self, current_file):
         except MemoryError as e:
             # cannot compute affinity propagation
             return None
+        except ValueError as e:
```

Comment:
When would a `ValueError` be raised?

Reply:
It was a suggestion from Ruiqing but it does not have any specific reason, so I removed it in my latest commit.